docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '3'
+
+services:
+  db:
+    image: mysql:5.7
+    environment:
+      MYSQL_ROOT_PASSWORD: my_secret_pw_shh
+      MYSQL_DATABASE: test_db
+      MYSQL_USER: devuser
+      MYSQL_PASSWORD: devpass
```

Comment:
As a noob in Docker, does this set up your database automatically?

Reply:
It does indeed, it does *not* import any data though.

FYI the way to look this sort of thing up is by trying to chase up the docs for the image. So the image is called `mysql`. if you look at https://hub.docker.com/_/mysql you can see some exposition of what the different environment variables do

Reply:
+1 this really helps, thank you!